themebuilder-scss/src/modules/compile-manager.ts

```diff
@@ -4,6 +4,7 @@ import PreCompiler from './pre-compiler';
 import resolveBundle from './bundle-resolver';
 import PostCompiler from './post-compiler';
 import BootstrapExtractor from './bootstrap-extractor';
+// eslint-disable-next-line
```

Comment:
What rule do we disable here?

Reply:
The module imported in the next line does not exist. It is generated while themebuilder is built.

Reply:
I suggest disabling a specific rule for this line. For instance:
https://github.com/DevExpress/DevExtreme/blob/5c77f06225bd4a73cd071eb1196d2868a69d7672/build/npm-bin/bundler.js#L14